Loop.xcodeproj/project.pbxproj

```diff
@@ -4118,7 +4134,7 @@
 		4F7528901DFE1DC600C322D6 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				APPLICATION_EXTENSION_API_ONLY = YES;
+				APPLICATION_EXTENSION_API_ONLY = NO;
```

Comment:
Was this an intentional change on the `Loop` project? If so, what was the reason?

Reply:
Yes.  I needed to do this in order to be able to call `UIApplication.shared.open()` in `NotificationsCriticalAlertPermissionsViewModel.swift`